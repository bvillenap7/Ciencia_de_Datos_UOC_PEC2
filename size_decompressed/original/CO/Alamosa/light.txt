Bright silverish-white light 5 degrees above horizon with occasional red flashes, appeared 50 miles North-West of Alamosa;1997-06-20 21:45:00;5 minutes;Summary : Bright silverish-white light 5 degrees above horizon with occasional red flashes, appeared 50 miles North-West of Alamosa Observed a bright silverish-white light about 5 degrees above the horizon directly over a straight line from US hwy 160.  I was about 5 miles west of Alamosa when object was first observed. It appeared to be aprox 50 miles NW of town  The light had occasional reddish flashes to it.  After about 5 minutes and after emerging from behind some trees as I was driving west bound the object was gone.;37.4725;-105.8796
Object like a star moving at a rapid rate from the south east.  Pin of white light stopped directly overhead, suddenly expanded blue.;1999-07-03 21:20:00;1 minute;Object like a star moving at a rapid rate from the south east.  Pin of white light stopped directly overhead.  Without warning it expanded into an orb of intense blue light which remained white in the center.  After about 5 seconds it vanished.  There was no sound. The size comparison, both at arms length, when first spotted would be that of an led light on a piece of electronic gear.  When it appeared to transform into an orb, the diameter would be that of looking into a standard flashlight.  A dramatic size increase in an instant.  My wife compared it to the color of a flame on a propane stove.  In retrospect, I think the object may have descended at an incredible rate of speed which would account for part of the size expansion.  My wife feels whatever it was may have been attracted by our small campfire.;37.4725;-105.8796
We saw three (or possibly four) orange random glowing lights bigger than stars or airplanes at 8:14 PM, March seventh, 2013.;2013-03-07 20:15:00;5-10 minutes;We saw three (or possibly four) orange random glowing lights bigger than stars or airplanes at 8:14 PM, March seventh, 2013. Hello, I’m currently a self-acclaimed UFOlogist (wish there was a college or something for it &#61514), and I’m not even sure what I saw.  I’ve read up a lot on this particular subject and I can say this (these) object(s) was truly unidentified.  I was riding in a car (minivan) with my mom, my little sister (nine), my little brother (nine months) and my two younger sisters in the back, both under five, fast asleep.  My mom has always kind of ridiculed my belief in UFOs (I’ve seen one before), so I at first thought she was joking when she said, I think I found your UFO. After she said she was serious several more times, I finally looked out a window, and saw nothing.  She described what she saw to me, she said she saw three little glowing orange lights ahead of her, much bigger than stars.   I looked out my window, and to my side I saw one as well.  We travel a lot very near to the Denver Airport so we both know what airplanes look like at night.  We live in rural Colorado so we both know what stars look like at night.  We were driving over La Veta pass at the time, and there weren’t many stars in the sky yet.   My first guess was airplane, but it looked absolutely nothing like an airplane, once I got a good look at it.   My second guess was ball lightning.  But there weren’t any clouds, and it hadn’t rained in a good week.  I was almost completely at loss. The more I thought the more I thought they were TRUFOs.  They disappeared one at a time, until they were all gone. I’ve seen helicopters at night, weather balloons, and all sorts of things.  I thought it might have been swamp gas but (Heh, heh) rural (not to mention dry) Colorado owns no swamps, and they didn’t look glowing.  I (luckily) remembered what I should do, though, and caught the time 8:15 PM, March seventh, 2013.  Please give me any helpful information to explain this! ((NUFORC Note:  Witness elects to remain totally anonymous provides little contact information.  PD));37.4725;-105.8796
