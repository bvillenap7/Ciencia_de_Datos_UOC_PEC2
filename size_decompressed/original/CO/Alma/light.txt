On Highway 9, right before you get into Alma going S.  Bright light, fast, greenish.  Only lasted a few seconds.;2016-10-20 19:35:00;5 seconds;On Highway 9, right before you get into Alma going South.  Bright light, fast, greenish.   Only lasted a few seconds before it went either behind treeline or disappeared.;39.2871;-106.0632
