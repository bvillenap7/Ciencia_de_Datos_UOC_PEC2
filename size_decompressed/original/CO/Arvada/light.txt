Two stars in the sky, moving and then dissappearing;1977-06-19 22:05:00;20 seconds;Two stars in the sky, moving and then dissappearing I looked in the southwest sky and saw two stars and almost immediately realized they were moving away from me. I checked their movement by looking at the other stars to make sure they weren't moving too. I watched the moving stars again and subsequently, they just faded away. There were at least three earthly crafts in the general viscinity as well.;39.8289982352941;-105.110888235294
Report to COMUFON: James Peters, 303-344-5576. Two observers. Very fast moving lights. Changes in direction and velocity.;1999-06-13 20:00:00;20 minutes;Report to COMUFON: ((name deleted)), 303-(telephone # deleted)). Two observers. Very fast moving lights. Changes in direction and velocity. At 20:00 hours, Observer1 notices a bright object from his back porch that is almost directly overhead.  Observer1 is sure of the time because he could hear a popular television program starting inside.  He was facing towards the south and the object was aproximately 80 degrees above the horizon, almost directly above his position.  The object catches his attention because of its brightness and that it seems to be moving around.  The object is not confused with normal air traffic, which was present at that time, planets or a satellite because of its intensity.  Observer1 calls his friend out to witness the object.  Observer2, who was a sniper in the Army, easily sees the object and three other objects in near proximity to it.  Observer1 goes inside to get his binoculars: West German, Steiner, Commander Pilot S, 7X50mm with a built in compass. With binoculars, Observer1 is able to see the other 3 objects noticed by observer2.  Observer1 is not able to make out some substance and colors to main object.  Main object begins to move rapidly to the south after loitering around with 3 smaller lights.  Observer1 tracks main object to the south while observer2 follows 3 smaller or less intense lights.  Observer1 watches the main, brightest object disappear beyond the horizon at a heading of 136 degrees in a time span estimated to be 5-8 seconds.  Observer2 sees other 3 objects depart straight up or radially from the Earth.  Observer1 not able to reaquire the 3 smaller lights and observer2 is able to see the 3 smaller lights longer than observer1.  Note: terms smaller and larger deal with the intesity of the lights and not necessarily to their true size.  The terms were used by the observer1 to report the event to this investigator.  The altitude of the objects seemed much higher than surrounding aircraft traffic.;39.8289982352941;-105.110888235294
Bright, stationary lights, began to move slowly the accelerated and vanished;2008-11-08 19:30:00;5 Min.;Bright, stationary lights, began to move slowly the accelerated and vanished Observed 2 bright, stationary lights above the area near 92nd and Wadsworth. Lights stayed staionary for several minutes, then began to move SSW very slowly. Slow movement continued for aprox. 1 minutes, when suddenly accelerated at an amazing speed and moved out of sight. Some commercial aircraft also in the area at the time. Helicopters seen in the vicinity later in the evening ((NUFORC Note:  Witness indicates that date of incident is approximate.  PD));39.8289982352941;-105.110888235294
What looked like a simple satellite, changed color, became more bright and expanded in 3-4 times in size.;2009-08-29 21:50:00;3 minutes;What looked like a simple satellite, changed color, became more bright and expanded in 3-4 times in size. Watching what I thought to be a satellite.  A small dim light, very high in the atmosphere moving steadily across the sky as I've witnessed other satellites do.  The light changed color to a vivid blue then brightened and expanded to at least 3-4 times the original size, all while moving at the same steady speed.  This lasted for about 10 seconds, then the light both dimmed and went back to it's original color.  I watched it for a few moments more, then it simply winked out.;39.8289982352941;-105.110888235294
Saw what looked like 2 gold flares over SE sky. Turned into 5 lights, stayed stationary then one by one the disappeared.  Then saw mult;2013-11-17 18:16:00;5 minutes;Saw what looked like 2 gold flares over SE sky.  Turned into 5 lights, stayed stationary then one by one the disappeared.   Then saw multi colored flashing light right over area.;39.8289982352941;-105.110888235294
Orange orb up in Lookout Mountain in Golden.;2014-11-28 19:50:00;20 minutes;Orange orb up in Lookout Mountain in Golden. My mother and I were on our daily drive to go visit my grandma, at the Arvada Cemetery.    As we were paying a visit to wish grandma a Happy Thanksgiving, we happen to notice a light that was not a regular part of the hillside.  It was seeming to float above the towers up on Lookout Mt. in Golden.   We watched the light,which was a orange amber colored oval shape light, dancing above the towers.  It moved and got lower to the horizon and shifted to the right.  And began to station itself above the third tower.   This was not the only thing this orb did, it grew and shrank in size, and would turn off like a light.  And reappear again.  It would vanish and reappear in the same spot.  After viewing it 20 minutes, it was gone! Like it just vanished in thin air!  I also went home, and got on the bike to see if I could investigate further.  I brought my highest viewing binos with me.  And to no avail, was not able to find the light that was over the towers on Lookout Mt.   I am trained and UFO investigator, since 2000.  I have many hours of field work.  And keep logs of what I see each year, and yearly on since today!  I have been seeing lots of these orange orbs here CO, and they seem to be apart of the night skies year after year!  But this orb was special to have seen.  It was the longest I have seen stay in one spot and disappear in thin air!   The temp outside was a 55 degrees and the moon was above the mountain as well.  Also, there was a few air traffic that flew above the orb, and that was when it turned off and then came back again after the plane had passed.  Which was very interesting to witness.  My mom and I were so amazed, we knew it was not something of our known knowledge.  So, we both said, UFO!  Couldn't identify it, and therefore was unknown to us.  Keep on looking Coloradans..  The skies keep getting more interesting over the years here!  Nights are fun to catch these orbs up in the foothills here in the Metro area.  Keep your eyes to the skies!;39.8289982352941;-105.110888235294
Amber Light Changes Solid Red.;2014-12-08 22:01:00;10 minutes;Amber Light Changes Solid Red Just went outside to bring the trash out in my neighborhood.  Happen to notice a amber gold light that was heading south towards Lakewood.  The light was right over my house here in town, and flew overhead above me.  This was not a commercial type plane or such.  As I watched the light/object continue on a southern heading, it began to take on another color.  It changed into a solid Red Ball!, about the size of  dime in distance!  The light now was a round ball of solid red light.  Not only did it change color, it was moving slower and began pulsating on and off!   I remember back in 2000 when my parents and I went down to visit the then first of it's kind here CO, The UFO Watchtower in the Valley!  That was the first time my father and I had seen such light of this nature!  It was just like the one we had seen down in Hooper in 2000.This Red Ball of light is always haunting to see! I always see this type of UFO here in CO.  Whatever it is,it is not ours.  And it is intelligent.  Tonight was not the first time I have seen the Red Ball of light.  During the earlier part of this year in Feb,I had witnessed several Red Balls of light.  In Feb., the lights were in a Triangle formation here in Arvada.  Heading West toward Golden.  I had made my report and contacted this site about it.I am just amazed how I can be in the right place at the right time to catch a good view of these lights.  Lucky I am.  I also have been reading Messages by Stan Romanek!  And he too, has seen lights just like the ones I see.  I guess this makes me wonder, what the heck are these lights doing?  And why here in CO period!!  Not sure!  But can say this,whatever they are,they move like nothing we have!  And disappear just as they appear as fast as they do!  I know I will keep looking up and keep on reporting to better help all other investigators in getting to the bottom of what these lights are.;39.8289982352941;-105.110888235294
Orange orb-like lights seen.;2019-07-10 20:50:00;15 minutes;Orange Orb like lights seen. On my way home from fishing, stopped at Stinger Field for a quick break to view the mountains.  Looking westwards,noticed multiple orange/yellow looking lights high up on the mountain range.  Lights were fading in and out!  Sometimes two appear at once, and then one again by itself!  These lights have been seen by me, almost every night when I come back from fishing.  During the winter time, the same lights can be seen as well.  The lights seem to be intelligent and controlled!  The lights reminds me of the orange orbs I used to see in Hooper and Center, in the San Luis Valley for many years now.  I will continue to report this summer and year of these orange lights seen here in Arvada and the Metro Area!  If any one else sees what I am seeing, I encourage you to give your details and report it here too! I am certain there are others out here who are seeing the same!  Also, last night on July 9th 2019 at the same time, the lights were seen along with a meteor!  It was a bright white fireball streak that broke up over the mountains where the lights were!;39.8289982352941;-105.110888235294
