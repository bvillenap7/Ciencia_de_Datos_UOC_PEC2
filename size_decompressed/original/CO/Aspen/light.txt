High object over Colorado moving erraticly, starting and stopping,  zigging and zagging, just before daybreak!;2006-11-17 05:30:00;30 minutes +;High object over Colorado moving erraticly, starting and stopping,  zigging and zagging, just before daybreak! My friend and I work the Graveyard shift.  We got off early (5:00 a.m.) and went for a soak in the hot tub.   Upon watching the beautiful clear sky, i soon thought that i had  spotted a sattelite.  It then proceeded to move, stop, reverse, turn, accelerate, and zig-zag all around but remained within the same basic airspace.   It looked very high (out of the atmospere) and i estimate its position to be about over Denver or Colorado Springs.   The color was that of a yellowish star, the luminosity was about like a medium star, and it looked bigger than the sattelites which began to appear at about first light (approximately 6:00).   The speed at which the object moved and changed course was phenominal and somewhat disturbing to me.   The random motions left me astonished and wondering, what are they doing up there?.   Thats no ((one word deleted))t, i promiss...;39.1977833333333;-106.825375
