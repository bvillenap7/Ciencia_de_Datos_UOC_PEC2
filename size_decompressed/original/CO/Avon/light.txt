Two solid white lights flying in formation over the Colorado Rockies;2006-07-13 11:00:00;1 minute;Two solid white lights flying in formation over the Colorado Rockies While looking at the stars, we saw two steady white lights flying in formation, traveling from north to south across the horizon.  They were travelling rather fast and at the exact same speed the entire time we saw them, one slightly below and behind the other.  The light went from similar to that of a star to very bright (resembling a spotlight) then gradually dimmer until we could no longer see them.  Almost as if the light was rotating very slowly.  When the light was brightest, it appeared to be vertically oblong in shape.  We waited to see if they appeared farther down the horizon, but they did not.  At the same time, airplanes were visible in other areas of the sky.  Since they looked nothing like the planes, we decided they must be UFOs.;39.6313;-106.5108
A very large bright light moving fromeast to west a a very high speed.;2009-07-22 21:30:00;30-40 seconds;A very large bright light moving fromeast to west a a very high speed. I was out walking my dog at about 9:30 at night.  I  live on top of a mountain at about  8,000 feet. I  looked up at the sky and saw a very large bright light moving at high speed from the east to the west.  There was no sound and no flashing lights.  It was not a plane.  I at first thought that it might be the space station.  However, I did some research on  the internet and the space station was not over Avon on that night.  There were also not any known satellites in the area.  It was also moving much too fast for the space station according to a friend who has watched it several times.  It was not a meteor and did not leave a trail. It moved horizontally and was extremely bright.  There have been many UFO sightings in our area recently.   I felt that it was important to add this one to your database.    I am also hoping that someone else saw this one.;39.6313;-106.5108
Two red lights over Avon, then 3 more over Eagle, CO.;2015-12-31 00:00:00;26 minutes;Two red lights over Avon, then 3 more over Eagle, CO. Left work and was driving west on I-70.  noticed two red lights up in the sky northside of the interstate in Avon.  About 26 minutes later, I was getting off at the Eagle exit and there were three more red lights up in the sky.  no motion or sound.;39.6313;-106.5108
2 aircraft w white lights early morning.;2016-11-17 07:45:00;2 minutes;2 aircraft w white lights early morning was out on the porch early morning and saw 2 unidentified aircraft each with 2 constant white lights (maybe reflections from the sun?) on each end fly erratically in and out of snow storm clouds. one stayed around longer flying in all directions very fast then vanished into the clouds. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6313;-106.5108
