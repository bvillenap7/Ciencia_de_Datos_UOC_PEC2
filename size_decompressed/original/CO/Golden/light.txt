I70E Golden, CO exit. Lg bright object sped to ground. Maybe a minute later I saw a bright flash of light(it crashed?);1997-06-10 22:30:00;2 minutes;I70E Golden, CO exit. Lg bright object sped to ground. Maybe a minute later I saw a bright flash of light(it crashed?) On June 10. 1997 my boyfriend and myself were heading east towards Denver, CO on I70.  we were just coming up to the exit for Golden (exit 261?) when I saw a large bright white object (maybe about the size of a car headlight if it was 1/4 mile away.)  It was going too fast to be a plane.  I don't think it was a metoerite or a shooting star.  It was coming from diretly ahead of us(east) and heading to the left (NW).  It was going at a 45 degree angle.  It looked like it crashed behind the town we could seee from the highway.  About a minute later I saw a flash of light coming from the same town.  The flassh was so bright that the mountain behind the town lite up.  It reminded me of the airport landing lightsssssss, but when I mentioned this to my boyfriend he said if it was airport lights they would repeat not just flash once and disappear.  I looked down at the clock and it was approx. 10:30pm. Unfortunately my bofriend didn't see it.  He thought I was pointing at something else.  The reason I noticed it was that it was a clear night and the object was very bright and moving very quickly.  I forgot to mention that it was round with no visible tail on it.  If we weren't in a hurry we would have gone to look for it.  For the next several days I looked in the papers to see if there was anything about it.  I also asked several people but no one heard of anything.;39.7598265306123;-105.220002040816
Blinking object entering atmosphere.;2001-11-29 18:55:00;3 seconds;Blinking object entering atmosphere. This is an extremely questionable occurance. I was waiting for a friend in a parking lot.  There was a ton of light pollution around.  Target, Safeway, lots of big shopping centers and stores was in the immediate area with huge parking lots. I looked up in the Northern sky.  I was on Colfax Avenue at Quail.  As I looked up, I spotted this ball light which was white around and an undetermined color in between the white.  This blinked and flashed as it must have entered the atmosphere.  However, it did not leave a tail behind!  This looked very very similar to a space craft entering the atmosphere but didn't see any recordings of occurance of this date and time especially anything heading northward! I was curious if there was another meteor shower going on, but there was no record that I could find of that either.  The reason I ask this, is that there were a couple other objects of same style of craft that must have entered the atmosphere.  Has anyone looked into this?;39.7598265306123;-105.220002040816
MUFON/COLORADO REPORT:   9 obj's that appeared to be bright hovering lights, then lit up, dimmed, moved to the E and began blinking.;2005-12-20 17:25:00;hour and a half on and of;9 objects that appeared to be bright hovering lights, then lit up, dimmed, moved to the east and began blinking. A friend and I were watching TV when around 5:27 he went out to the balcony of my apartment for a smoke.  I live up on a hill on the border of Golden and Lakewood that overlooks Denver and its suburbs to the north.   He knocked on the window for me to come and look at something.  I am a MUFON Investigator so I watch the skies often.  He pointed out a bright greenish light in the sky to the NNE.  It was just sitting there perfectly still.  I thought it looked very strange so I grabbed my binoculars.  He then pointed out that there were actually 4 objects.  All of them were still.   That looked even more interesting so I grabbed my video camera.  Once I got out there with the camera they began moving.  When I got my camera on them there were three left.  The forth had moved off to the east.  When I began video taping one object began to move off to the east, and one to the west.  The one to the west then came back moving to the east.  Then while filming the remaining light, which was perfectly still, just its right and above it another light lit up.  Soon after, the bottom light lit up very brightly.  When it got very bright, it appeared like an angled disk shape.  It then began to move to the east like the rest and dim out.  All of them did this when they flew easy and as they got dimmer they also began to blink.  The remaining light did the same thing.  It began to get extremely bright, and then it began moving east.   Here again it would get dimmer and begin to blink.  This can all be seen on the tape.  As it moved to the east, I noticed yet another light appear the same place the last one had appeared.  This one was dimmer and began move to the east blinking immediately.  When the tress got in the way I stopped filming.  However, we could see at least two of the light off in the distance over Denver.  Then after a few minutes they began to fade too dim to see.  At this point I called the Director of Investigations to let him know what we saw.  That call was made at about 5:55. While on the call at about 6:15, my friend could see another light appearing in the same spot they had appeared before.  I got off the phone and went back with the camera, however right when I got the camera on it, it faded very dim.  My friend saw another one of them over Denver that was lighting up very bright again.  Again once I got the camera on it, it faded out.  Because I thought they might be reacting to the camera I put it away.  A couple minutes later another light appeared where the others had.  This time, this one stayed still.  It looked like it actually was two light side by side.  As it sat there another light began to light just below it to the left.  The light appearing looked very disk shaped through the binoculars.  After 30 seconds or so as it finally seemed to be finished appearing or whatever it was doing and they both moved to the east.  One headed east over Denver, the other seemed to keep a regular distance from us and moved in an arc to the east with us in the middle.  Once it got due south of us, it just seemed to hover out there.  We then noticed one out over Denver.  This one seemed to be east of downtown and it had two lights, one at either end.  It moved south and appeared very large for as far away it was.  I would say about half the size of a dime at arms length.  The light towards the front was brighter, and eventually the light at the back began to pulse.  It eventually dimmed out.  We could see one of the light heading east still, but after a couple minutes it began to get too dim also.  At this point the light to the south was no longer visible either.  In total we saw nine of these objects and got seven on tape.  You can look for yourself, but it does not appear that these objects had the characteristics of any known craft.  The closest would be a helicopter, but that would not explain why some appeared to come out of no where, and why they seemed solid at first.  It also would not explain the apparent disk shape.  Two other things I would like to note, although they seem extraordinary is that they seemed to react to being video taped, and they seemed to intentionally blink to mimic airplanes. ((NUFORC Note:  We express our sincere gratitude to this MUFON member for submitting such an excellent report.  This is one of two reports submitted by this individual, over the last three years.  PD));39.7598265306123;-105.220002040816
There were 7 vivid, gark green star like lights in the sky in the southwest direction.  ((NUFORC Note:  Stars??   PD));2015-12-14 21:00:00;3 hours;There were 7 vivid, gark green star like lights in the sky in the southwest direction. I was in my bed at home, around 9:00.  I looked out the window (facing west) and noticed some strange lights, there we're 4 lights that looked like stars in the south west direction. However, these were no stars seeing as these lights were a vibrant, dark, green.    I looked it up and saw that it is impossible for a star to be green, plus not all of them were being covered by clouds.  3 out of the 4 had disappeared, maybe by clouds?  So now afraid that I was running out of time I asked my sister to come up stairs and look at them.  She confirmed the light and as she was looking I saw 3 more of the same lights exept these ones were closer to the horizon and dawned just above a roof top.  These ones are a lot harder to spot than the first one I saw from the original 4.   Again, these look like stars ans twinkle but are a green color and not all are being covered by clouds. ((NUFORC Note:  We suspect that the witnesses may have been looking at twinkling stars.  PD));39.7598265306123;-105.220002040816
there were two orange lights coming from over Lookout Mountain antennas. They made no sounds,  they moved towards us then stopped then;2017-05-22 23:18:00;3 minutes;there were two orange lights coming from over Lookout Mountain antennas.  They made no sounds, they moved towards us then stopped then moved together then one slowly moved up the other down and then disappeared there was no cloud cover for them to disappear in.;39.7598265306123;-105.220002040816
Blue/white Orb spotted on Lookout Mountain;2017-12-02 01:00:00;1 second;blue/white Orb spotted on Lookout Mountain We drove from Boulder to Lookout Mountain.  We arrived at our spot at 39°43'51.9N 105°14'28.4W. (Facing West)  We sat there listening to music for a couple hours, when all of the sudden out of the corner of my eye I saw a white/blue light hovering between two trees (it was not the reflection of car headlights, because those reflections always followed the same horizontal pattern, when I looked at the light it was just hovering there.)  I looked to see what this light was that caught my eye, as did the person sitting next to me.  Together we saw an orb.  It looked to be about 100 feet from the vehicle.  As soon as we looked at it, it zipped straight down behind the hill.  We didn't see it again.;39.7598265306123;-105.220002040816
